db/init_db.js

```diff
@@ -57,206 +57,209 @@ async function buildTables() {
         quantity INTEGER NOT NULL DEFAULT 0
       );
     `);
+    console.log("Finished building tables.")
+  } catch (error) {
+    
+  }
+  
 }
 
+//map calls the callback func once for each element (good), in order (not necessary),
+//+ constructs new array from results
+
 async function createInitialUsers() {
-  console.log("Creating Users ...");
   try {
+    console.log("Creating Users ...");
     // create useful starting data
-    const initialUsers = [
+    await createUser(
       {
         username: "beatricewhite",
         password: "111111",
         firstName: "Beatrice",
         lastName: "White",
         email: "beatricewhitee@gmail.com",
         isAdmin: "false",
```

Comment:
Do these isAdmin values need to be strings ("true"), or booleans (true)?

Reply:
They would be booleans! nice catch!!